src/message.ts

```diff
@@ -667,12 +858,34 @@ export class Message implements Sayable {
 // Message.initType()
 
 export class MediaMessage extends Message {
+  /**
+   * @private
```

Comment:
I think if the class property is already `private`, then we will not need to use `@private` in jsdoc again?